python/modelUtils.py

```diff
@@ -9,10 +9,11 @@
 from tensorflow.keras import layers
 import tensorflow.keras.backend as K
 from sklearn.model_selection import train_test_split
+from lrscheduler import get_lr_scheduler
 
 import plotter
 
-n_models_in_parallel = 1
+n_models_in_parallel = 5
```

Comment:
Let's keep the default to 1.

Reply:
thanks for pointing this out. I just keep forgetting about this

Reply:
At the moment there is only this one setting for model configs. I'll note this down and implement it later. It will be convenient to write custom model structures directly in model_config